test/blackbox-tests/test-cases/foreign-library.t/run.t

```diff
@@ -91,6 +91,7 @@ Testsuite for the (foreign_library ...) stanza.
   >  (flags -DCONFIG_VALUE=2000)
   >  (names config))
   > EOF
+  $ touch lib/calc.ml
```

Comment:
Is this an accidental change?

Reply:
No, it is another case where we started reporting more errors than before. Here, Dune was complaining about `calc.ml` not existing. Since that wasn't the point of the test, I thought it was cluttering the output and added this line to remove the error.

Reply:
I see, interesting.